examples/lock-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -39,8 +39,9 @@ using namespace ::chip::DeviceLayer;
 void AppDeviceCallbacks::PostAttributeChangeCallback(EndpointId endpointId, ClusterId clusterId, AttributeId attributeId,
                                                      uint8_t type, uint16_t size, uint8_t * value)
 {
-    ESP_LOGI(TAG, "PostAttributeChangeCallback - Cluster ID: '0x%04x', EndPoint ID: '0x%02x', Attribute ID: '0x%04x'", clusterId,
-             endpointId, attributeId);
+    ESP_LOGI(TAG,
+             "PostAttributeChangeCallback - Cluster ID: '0x%04" PRIx32 "', EndPoint ID: '0x%02x', Attribute ID: '0x%04" PRIx32 "'",
```

Comment:
Pre-existing, but this code seems to think that cluster/attribute ids are 16-bit and endpoint ids are 8-bit....

Reply:
Yeah, fixed these